src/pages/public_registration/ActionPanelContent.tsx

```diff
@@ -53,6 +61,33 @@ export const ActionPanelContent = ({
   const isInRegistrationWizard =
     window.location.pathname.startsWith(UrlPathTemplate.RegistrationNew) && window.location.pathname.endsWith('/edit');
 
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const deleteDraftRegistration = async () => {
```

Comment:
For nye ting kunne vi med fordel brukt `useMutation` i stedet for sånne egne custom click-handlers da

Reply:
Da slipper vi å håndtere retry, loading, og response-kode selv